test/opexecuter.cpp

```diff
@@ -409,9 +397,13 @@ void OpExecuter::ExecuteOnNGraph(vector<Tensor>& ngraph_outputs,
   try {
     backend->call(backend->compile(ng_function), ng_op_tensors, ng_ip_tensors);
   } catch (const std::exception& exp) {
+    BackendManager::UnlockBackend(ng_backend_type);
+    NgraphSerialize("unit_test_error_" + test_op_type_ + ".json", ng_function);
     std::cout << "Exception while executing on nGraph " << exp.what()
```

Comment:
The execution must stop at this point i.e., once the exception is encountered, this function should re-throw the previous exception. This is an existing bug - but please fix this in this PR.